src/app/settings/user/template.html

```diff
@@ -38,6 +38,14 @@
            fxLayoutGap="10px">
         <div class="user-name">{{user.name}}</div>
         <div class="user-email">{{user.email}}</div>
+        <km-property *ngIf="user.groups">
```

Comment:
Inorder to show empty placeholder we need to remove this `*ngIf`

Expected output in case no groups exists.

![display empty placeholder](https://user-images.githubusercontent.com/17727069/187378724-66d96fe9-9aff-420f-916a-60fbfc2a8d77.png)

Reply:
Actually, I'm hesitating if we should show the OIDC groups if not configured? @cschieder WDYT?